setup.py

```diff
@@ -58,19 +100,17 @@ def build_extension(self, ext):
                                                               self.distribution.get_version())
         if not os.path.exists(self.build_temp):
             os.makedirs(self.build_temp)
-        
+
         cmake_args += ['-DUSE_PYALEMBIC=On']
-        subprocess.check_call(['cmake', ext.sourcedir] + cmake_args, cwd=self.build_temp, env=env)
-        subprocess.check_call(['cmake', '--build', '.'] + build_args, cwd=self.build_temp)
-	
-        # Copy missing dependencies
-        site_dir = "/usr/local/lib/python2.7/site-packages"
-        missing_packages = ["iexmodule.la", "iexmodule.so", "imathmodule.la", "imathmodule.so", "imathnumpymodule.la", "imathnumpymodule.so"]
-        for package in missing_packages:
-            shutil.copyfile(os.path.join(site_dir, package), os.path.join(extdir, package))
+        subprocess.check_call(
+            [cmake_command, ext.sourcedir] + cmake_args, cwd=self.build_temp, env=env)
+        subprocess.check_call(
+            [cmake_command, '--build', '.'] + build_args, cwd=self.build_temp)
+
+
 setup(
     name='alembic',
-    version='1.7.16',
+    version='1.8.2',
```

Comment:
this should most likely not be hard coded but should contain the current tag name for tags and the short hash for commits? e.g. something like https://stackoverflow.com/questions/58177786/get-the-current-pushed-tag-in-github-actions

Reply:
Thanks Manne - I will open a new PR with both python2.7 and python3.7 manylinux wheel builds.

As pyalembic can also be built from source locally, I'm unsure of the best approach towards automated versioning. Additionally, github actions currently doesn't support a simple way to pass build-args to Docker, which would need access to these environment variables.

Currently, the values are hard coded in the CMakeLists.txt files - maybe we could extract the version from there?